cvmfs-mon/cvmfs-mon.py

```diff
@@ -42,7 +42,7 @@ def check(monit):
     try:
       api_url = monit["api_url"] + "/" + repo
       s = get(api_url).json()
-      ok = s["status"] == "ok"
```

Comment:
What is the point of the comparison in the first place? Please split also this one.